### PR TITLE
Run 2 enums from DataTypes

### PR DESCRIPTION
**Pull request by ddobrigk**
@marcovanleeuwen, @mconcas, @qgp: @jgrosseo pointed out that the Run2 utils header for the ALICE3 AO2D conversion tool actually re-declared some enums that already exist in DataTypes.h, so to avoid confusion, I just re-used those and removed the double declarations.

### CODE REVIEW

**Review by mconcas — DISMISSED** [review]

**Comment by ddobrigk** [comment]
Marking as draft pending the merging of a rename PR from earlier today from @jgrosseo - will require minor changes after that one is merged

**Comment by ddobrigk** [comment]
Now compatible with [this PR](https://github.com/AliceO2Group/AliceO2/pull/5843)

**Comment by ddobrigk** [comment]
Hi everyone, this is just a friendly ping since this was popping up just before the easter break :-D I think it still needs approval from either @qgp or @marcovanleeuwen (not sure why @mconcas' approval is no longer good for the system... but ok)

**Comment by mconcas** [comment]
> Hi everyone, this is just a friendly ping since this was popping up just before the Christmas break :-D I think it still needs approval from either @qgp or @marcovanleeuwen (not sure why @mconcas' approval is no longer good for the system... but ok)

Hi David, could you please fix the errors raised by the CI? Thanks in advance.
PS: when you push other commits after an approval it automatically dismisses it as the code actually changed.

**Comment by ddobrigk** [comment]
> > Hi everyone, this is just a friendly ping since this was popping up just before the Christmas break :-D I think it still needs approval from either @qgp or @marcovanleeuwen (not sure why @mconcas' approval is no longer good for the system... but ok)
> 
> Hi David, could you please fix the errors raised by the CI? Thanks in advance.
> PS: when you push other commits after an approval it automatically dismisses it as the code actually changed.

Oooops, sorry, my bad - forgot an include! Now fixed. 

About your P.S.: Sure, that makes sense. My comment was more regarding the fact that git says you cannot approve anymore (but previously you could) as it only mentions @qgp and @marcovanleeuwen as approvers now. No issue in any case for me of course :-D

**Review by mconcas — APPROVED** [review]

**Comment by ddobrigk** [comment]
Hi everyone, is there something still missing before we can merge this PR?... Sorry if I miss something obvious.

**Pull request merged**
